umm-spec-lib/resources/json-schemas/variable/umm/v1.8/business_rules_v1.8.md

```diff
@@ -0,0 +1,14 @@
+## Changes for 1.8
+
+(This is accompanying ancillary documenation that is useful for CMR and MMT to handle the latest implementation of schema changes.  These include any noteworthy ingest/field migration rules, how the MMT forms should handle the new/additional fields, how the CMR preview should handle the new/additional fields, etc.).
+
+### CMR
+- Migrating UMM-Var records up to 1.8 add the MetadataSpecification for version 1.8
+  Migrate down to 1.7 from 1.8 drop MetadataSpecification drop RelatedURLs.
```

Comment:
wording for the end of the sentence is odd, does it drop both MetadataSpecificiation and RelatedURLs?

Reply:
yes, but I merged already. Will correct in my next change.